src/main/java/uk/gov/hmcts/darts/cases/exception/CaseApiError.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.cases.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum CaseApiError implements DartsApiError {
+
+    COURTHOUSE_PROVIDED_DOES_NOT_EXIST(
+        "100",
+        HttpStatus.BAD_REQUEST,
+        "Provided courthouse does not exist"
+    ),
+
+    COURTROOM_PROVIDED_DOES_NOT_EXIST(
```

Comment:
I'm not sure this error makes sense? I think if we don't have the courtroom already it should be created.

Reply:
I can see that it was implemented in the case of getCases, I can implement it for addCase.

Reply:
There's something in the common/api package for get or create courtroom, can we use that?

Reply:
Yes there is a method called retrieveOrCreateCourtroom.